src/qt/bitcoingui.cpp

```diff
@@ -123,7 +123,11 @@ BitcoinGUI::BitcoinGUI(const PlatformStyle *_platformStyle, const NetworkStyle *
     spinnerFrame(0),
     platformStyle(_platformStyle)
 {
-    GUIUtil::restoreWindowGeometry("nWindow", QSize(850, 550), this);
+    QSettings settings;
```

Comment:
Honestly I think it's better to keep `restoreWindowGeometry` and move this there.

Reply:
Same as below, I think its a lot cleaner inline, will change if others agree though :+1: